packages/typespec-client-generator-core/test/package.test.ts

```diff
@@ -3091,17 +3055,429 @@ describe("typespec-client-generator-core: package", () => {
       op test(...Intersected): void;
       `);
       const method = getServiceMethodOfClient(runner.context.experimental_sdkPackage);
-      const documentMethodParam = method.parameters.find(
-        (x) => x.name === "documentTranslateContent"
-      );
+      const documentMethodParam = method.parameters.find((x) => x.name === "document");
       ok(documentMethodParam);
       strictEqual(documentMethodParam.kind, "method");
       const op = method.operation;
       ok(op.bodyParam);
       strictEqual(op.bodyParam.kind, "body");
-      strictEqual(op.bodyParam.name, "documentTranslateContent");
+      strictEqual(op.bodyParam.name, "testRequest");
       deepStrictEqual(op.bodyParam.correspondingMethodParams, [documentMethodParam]);
     });
+
+    it("anonymous model with @body should not be spread", async () => {
```

Comment:
Do we have a case for alias? I believe it is the same as spread model?

```
alias Test = {
  prop: string
};
op test(...Test): void;
```

Reply:
How about this one? Also spread case, right?

```
model Test {
  @header
  foo: string;
  prop: string
};
op test(...Test): void;
```

Reply:
same as spread model, i think it has been covered by another case.

Reply:
What I take is 
1. if there is `@body` or `@bodyRoot` anywhere, it is explicit body, and it is not spread
2. otherwise, spread